mlflow/resource_experiment.go

```diff
@@ -0,0 +1,160 @@
+package mlflow
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// MLFLowExperiment defines the parameters that can be set in the resource.
+type MLFLowExperiment struct {
```

Comment:
Remove MLflow prefix from all of entities - it's clear from package name it's MLflow